tests/test_03_3Projects_ExportImport3.py

```diff
@@ -27,6 +27,122 @@ def test_main(self):
     """
     main function
     """
+    ###### BEGIN testELNFile: copy into all export tests in this folder   ######
+    def testELNFile(fileName:str, verbose:bool) -> None:
```

Comment:
Why do we duplicate the same function in both classes, is there any specific reason to do so? I would go for the previous implementation itself, importing testELNFile from tools!

Reply:
@jmurugan-fzj I tried that but then I get a fixtures error. I tried all morning to understand / wrap my head around fixutures. I asked chatGPT. But nobody could help.